src/main/scala/zio/prelude/NonEmptyList.scala

```diff
@@ -64,6 +64,35 @@ sealed trait NonEmptyList[+A] { self =>
         if (seen(hash)) (as, seen) else (cons(a, as), seen + hash)
     }._1.reverse
 
+  /**
+   * Drops the first `n` elements from this `NonEmptyList` returning a `List`.
+   */
+  @tailrec
+  final def drop(n: Int): List[A] =
+    if (n <= 0) self
+    else
+      self match {
+        case Cons(_, t) => t.drop(n - 1)
+        case Single(_)  => Nil
+      }
+
+  /**
+   * Drops the last `n` elements from this `NonEmptyList` returning a `List`.
+   */
+  final def dropRight(n: Int): List[A] =
+    reverse.drop(n).reverse
```

Comment:
I think `take(length - n)` is equivalent and would be slightly more efficient (one traversal for `length` and another for `take` versus two travels for `reverse` and one for `drop`).

Reply:
Done.